soco/music_services/music_service.py

```diff
@@ -153,12 +145,7 @@ def get_soap_header(self):
                 token.text = token_pair[0]
                 key.text = token_pair[1]
 
-            household_id = XML.SubElement(login_token, "householdId")
-            household_id.text = self._household_id
-            credentials_header.append(login_token)
-
-        # TODO Implement UserID with user provided account, since we can't get the
-        # accounts from the device anymore
+                credentials_header.append(login_token)
```

Comment:
This change breaks Amazon Music auth in my tests.

Reply:
This should not have been removed... It still part of my code...

Reply:
Specifically the removal of the `householdId` subelement breaks it.